docs/training_guides/getting_set_up.rst

```diff
@@ -0,0 +1,172 @@
+
+You can use the ``mkdir`` command to create a new directory, ie: ``mkdir home/username/virtualenvs``. We'll refer to the full path to this directory as **$path_to_your_env** below.
+
+The next step is to create your virtual environment within this directory. The commands are different based on whether you're on Python 3.4+ and using venv, or whether you're using an older version with the virtualenv program you just installed.
+
+**If you’ve got Python 3.4 or higher**, type ``python -m venv $path_to_your_env/$your_env_name``. The path should be the directory you created to store the virtual environments, and the environment name is a new name chosen by you.
+
+**If you’ve got a lower Python version**, type ``virtualenv $path_to_your_env/$your_env_name``. Again, the path should be the directory for storing virtual environments, and the env name is a new name.
+
+Regardless of what version you're on, you can activate your virtual environment with the command: ``source $path_to_your_env/$your_env_name/bin/activate``.
+
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Windows Virtual Environments
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Start by installing virtualenvwrappers from source::
+
+  	git clone git://github.com/davidmarble/virtualenvwrapper-win.git
+  	cd virtualenvwrapper-win
+  	python setup.py install
+
+Find the Scripts\ directory for your Python installation, such as ``C:\Users\<User>\AppData\Local\Programs\Python\Python37\Scripts\``.
+
+Add the Scripts\ directory to your $PATH. (Not sure what a $PATH is?  see :ref:`Paths vs $PATHs<path-explainer>`.)
+
+To create a virtual environment for Parsons, execute: ``mkvirtualenv parsons``
+
+To use this virtual environment, execute: ``workon parsons``
+
+.. _path-explainer:
+
+^^^^^^^^^^^^^^^
+Paths vs $PATHs
+^^^^^^^^^^^^^^^
+
+Paths are how we refer to a file or program's location in the file system. For example, ``home/janedoe/virtualenvs`` says that within the top-level directory ``home`` there is a directory named ``janedoe``, and within ``janedoe`` there is a directory ``virtualenvs``.
+
+``home/janedoe/virtualenvs`` is an **absolute path** because it specifies exactly how to get there no matter where you are in the system. The path ``janedoe/virtualenvs`` is a **relative path** because it only works if you use it from the home directory. Trying to use a relative path from the wrong location is a common source of command line errors!
+
+In these instructions we try to use absolute paths, even though they're a little wordier, because it's less likely to cause problems for you if you run them from an unexpected place.
+
+Confusingly, there's an important environmental variable called **$PATH**. The $PATH is a list of places your computer will check when searching for installed libraries and scripts. You can check what's currently in your $PATH by typing ``echo $PATH`` (Mac/Linux) or ``echo %PATH%`` (Windows).
```

Comment:
I don't think we should say it's confusing. I think it's generally good practice to be empowering in documentation, particularly for beginners.

Reply:
Do you see that wording as disempowering? My instinct is that people find it empowering to hear that other people find confusing things confusing, but OTOH if I'm the only one who finds path vs $PATH confusing I definitely don't want to make others doubt themselves.

Reply:
Yeah, I think so. I think you're right - it is a confusing concept. I think a good analogy is GotV literature. We know that voting in a lot of places is confusing and difficult to do properly. But it's been found that it's important to have a positive message about voting ("Voting is easy", "Voting is fun", etc) even if we don't believe it, because people won't vote if we're telling them it's difficult. Even if we're also telling them how to do it.

In any case, I don't really think it's _that_ important - it's just one word. I'll defer to you, but did want to share my opinion.

Reply:
No, you've convinced me. :)  I changed the wording.